sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -354,12 +336,11 @@ private async Task ServiceBusEndToEndInternal<T>(IHost host)
             IEnumerable<LogMessage> logMessages = host.GetTestLoggerProvider()
                 .GetAllLogMessages();
 
-            // filter out anything from the custom processor for easier validation.
-            IEnumerable<LogMessage> consoleOutput = logMessages
-                .Where(m => m.Category != CustomMessagingProvider.CustomMessagingCategory);
-
-            Assert.False(consoleOutput.Where(p => p.Level == LogLevel.Error).Any());
+            //filter out anything from the Azure SDK (Service Bus, Identity, Core) for easier validation.
+           IEnumerable <LogMessage> consoleOutput = logMessages
+               .Where(m => !m.Category.StartsWith("Azure."));
 
+           Assert.False(consoleOutput.Where(p => p.Level == LogLevel.Error).Any());
```

Comment:
Should we filter after checking for errors?

Reply:
Added a comment to explain why this is done.